### PR TITLE
Add headbutting open airlocks when you have no hands

### PR DESCRIPTION
**Pull request by DrSmugleaf**
bang.ogg sound is from https://github.com/tgstation/tgstation/blob/3af0c4e0b883add51d0719459c05cc6f8d77abc8/sound/effects/bang.ogg

For Abraham:
image

### CODE REVIEW

File: Content.Server/GameObjects/Components/Doors/ServerDoorComponent.cs [hunk 1]
```diff
@@ -145,7 +149,14 @@ public void TryOpen(IEntity user)
                 Deny();
                 return;
             }
+
             Open();
+
+            if (user.TryGetComponent(out HandsComponent hands) && hands.Count == 2)
```
Comment (PJB3005) [inline Content.Server/GameObjects/Components/Doors/ServerDoorComponent.cs:155]: > `hands.Count == 2`
> no hands

:thinking:

Comment (Zumorica) [inline Content.Server/GameObjects/Components/Doors/ServerDoorComponent.cs:155]: ```hands.Count == 2``` :thinking:

Comment (DrSmugleaf) [reply Content.Server/GameObjects/Components/Doors/ServerDoorComponent.cs:155]: Gazoinks, you have caught my trick to record the video.

**Comment by unusualcrow** [comment]
missed opportunity https://www.youtube.com/watch?v=gwxTZaa3NgI

**Pull request merged**